requirements/default.txt

```diff
@@ -1218,10 +1218,6 @@ sacrebleu==2.3.1 \
 sacremoses==0.0.43 \
     --hash=sha256:123c1bf2664351fb05e16f87d3786dbe44a050cfd7b85161c09ad9a63a8e2948
     # via -r requirements/default.in
-setuptools==75.1.0 \
```

Comment:
This got in with your last update, but I'm not sure how/why.

`make requirements` removes it for me, and I'm not sure why yours added it.